README.md

```diff
@@ -1,17 +1,31 @@
 # hello_microverse
-**Hello Microverse Project** is a simple exercise to demonstrate my understanding on the following:
-github flows, 
-linters, 
-github pull requests,
-and my ability to submit a project for code review
+> Hello Microverse Project is a simple exercise to demonstrate my understanding on the following: github flows,linters, github pull requests, and my ability to submit a project for code review
 
-# Author:
-Joshua Blue-Jack
-linkedin: Joshua Blue-Jack
-github: https://github.com/Tamunokuro
+# Built with
+- HTML and CSS
+- Linters
+
+# Getting Started 
+** Clone the repository using git clone (https://github.com/Tamunokuro)**
+
+## Prerequisites
+- Code Editor
+- GIT
+- NODE PACKAGE MANAGER
+
+# Authors
+## 👤 Authors
+
+- Linkedin: (www.linkedin.com/in/joshua-blue-jack/)
+- Github: (https://github.com/Tamunokuro)
```

Comment:
[Optional] You can add links to your online profiles using this syntax `[Link Title](Link URL)`.